package.json

```diff
@@ -96,7 +81,7 @@
   "dependencies": {
     "@aws-sdk/client-sqs": "^3.596.0",
     "@ministryofjustice/frontend": "^2.1.3",
-    "@opentelemetry/api": "~1.9.0",
+    "@opentelemetry/api": "~1.8.0",
     "agentkeepalive": "^4.5.0",
```

Comment:
too bad there's no easy write to put a comment on this to remind me not to update this package.

Reply:
I have often wished for the same for similar reasons!

Reply:
there are suggested workaround such as:
```
    "@opentelemetry/api": "// keep this at 1.8.0 to maintain compatibility with other dependencies",
    "@opentelemetry/api": "~1.8.0",
```

but i wouldn't want to tinker with how this weirdness will interact with renovate and everything else